Core/CoreTests/SubmitAssignmentExtension/Model/AttachmentSubmissionServiceTests.swift

```diff
@@ -134,11 +136,12 @@ class MockFileSubmissionAssembly: FileSubmissionAssembly {
 class MockFileSubmissionComposer: FileSubmissionComposer {
     var deletedSubmission: NSManagedObjectID?
     var deletedItem: NSManagedObjectID?
-    var startedSubmission: (courseId: String, assignmentId: String, comment: String?, files: [URL])?
+    // swiftlint:disable:next large_tuple
+    var startedSubmission: (courseId: String, assignmentId: String, assignmentName: String, comment: String?, files: [URL])?
```

Comment:
We could probably introduce a new structure for this like `StartedSubmissionParams`

Reply:
Agreed.